PublishParasoftResults/xsl/sarif.xsl

```diff
@@ -213,6 +213,7 @@
         <xsl:call-template name="severity_level">
             <xsl:with-param name="parsoft_severity" select="@sev"/>
         </xsl:call-template>
+        <xsl:text>, "violType": "</xsl:text><xsl:value-of select="name()" /><xsl:text>"</xsl:text>
```

Comment:
There is warning when using VScode TO open sarif report.
![image](https://github.com/parasoft/parasoft-findings-vsts/assets/109122571/7b78792b-12af-46d6-8e05-fda7ff1bef34)

Move it to "partialFingerprints" section.